doc/api/module.md

```diff
@@ -269,65 +164,225 @@ changes:
                  `globalPreload`; added `load` hook and `getGlobalPreload` hook.
 -->
 
-> Stability: 1 - Experimental
-
-> This API is currently being redesigned and will still change.
+> Stability: 1.1 - Active development
 
 <!-- type=misc -->
 
-To customize the default module resolution, loader hooks can optionally be
-provided via a `--experimental-loader ./loader-name.mjs` argument to Node.js.
+<i id="enabling_module_customization_hooks"></i>
+
+### Enabling
+
+Module resolution and loading can be customized by registering a file which
+exports a set of hooks. This can be done using the [`register`][] method
+imported from `node:module`, which you can run before your application code by
+using the `--import` flag:
+
+```bash
```

Comment:
We seem to use both in our docs, `bash` more so than `console`.

Reply:
Then we have many places that are wrong 🤷‍♂️ In this case correctness is better than consistently wrong, no?

Bash is for a shell file (like `foo.sh`)

Console is for a CLI prompt: https://stackoverflow.com/a/49004070/758177

Reply:
Well that's `bash`, not `console`. `console` is when you see console output, `bash` is for shell script that you can copy paste to your terminal (or bash script)

Reply:
From the examples you just gave, wouldn't you agree that `bash` is more suited than `console` here?

Reply:
We _happen_ to not see a difference, but if the CLI snippet changes, it could incorrectly/needlessly break, and the editor may not know why.

I'm not saying "this absolutely must be fixed"; I'm saying it's wrong and easily fixed to avoid easily avoidable problems in future.

Reply:
We'll have to agree to disagree here, I would feel strongly that `console` is wrong here, the code snippet _is_ a `bash` command. I've spent quite some time on https://github.com/nodejs/node/commit/260092eca316bae6036e52b2cdc73c4c4f8a9da6 to make sure the doc were consistent on that matter.

Reply:
I’m going to mark this as resolved and you can open a new PR for this, perhaps to make it consistent throughout the codebase and enforced via lint rules. The existing examples used/use `bash` so I’m going to leave it that way for now.